examples/gradle-kt-dsl/lib/src/test/kotlin/com/amazon/corretto/crypto/examples/AesGcm.kt

```diff
@@ -0,0 +1,42 @@
+package com.amazon.corretto.crypto.examples
+
+import com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider
+import java.nio.charset.Charset
+import java.security.SecureRandom
+import javax.crypto.Cipher
+import javax.crypto.spec.GCMParameterSpec
+import javax.crypto.spec.SecretKeySpec
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+fun SecureRandom.randomSeq(len: Int): ByteArray {
```

Comment:
huh, that's kind of neat that kotlin lets you define methods on existing classes in-line like this